docs/source/keywords.rst

```diff
@@ -734,6 +734,11 @@ Kokkos will attempt to autodetect the architecture flag at configuration time.
 Intel GPUs
 ~~~~~~~~~~
 
+If the SYCL backend is enabled and no GPU architecture is specified,
+Kokkos will use Just-In-Time (JIT) compilation and select the first visible device for the default execution space instance.
```

Comment:
Although the discussion about device selection is important, I am not sure it belongs here

Reply:
Yeah this needs to go to SYCL somewhere. Not in the architecture selection.

Reply:
Since I can't find a better place and you feel strongly about not having it here, I just dropped the note.

Reply:
@crtrott suggested putting it here https://kokkos.org/kokkos-core-wiki/API/core/execution_spaces.html#kokkos-sycl since it is more sycl related than Intel GPU architecture related